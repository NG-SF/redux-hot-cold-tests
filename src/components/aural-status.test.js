import React from 'react';
import {shallow} from 'enzyme';
import renderer from 'react-test-renderer';
import {AuralStatus} from './aural-status';

describe('<AuralStatus />', () => {
  it('Renders without crashing', () => {
    shallow(<AuralStatus />);
  });

  it('Should match its snapshot', () => {
    const tree = renderer.create(<AuralStatus />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Renders status update', () => {
    let update = 'Status update';
    let wrapper = shallow(<AuralStatus auralStatus={update} />);
    expect(wrapper.contains(update)).toEqual(true);
  });
});