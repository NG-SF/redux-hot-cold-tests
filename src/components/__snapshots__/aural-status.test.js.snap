// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AuralStatus /> Should match its snapshot 1`] = `
<p
  aria-atomic="true"
  aria-live="assertive"
  className="visuallyhidden"
  id="status-readout"
/>
`;
